meta/main.yml

```
@@ -1,11 +1,12 @@
 ---
 dependencies: []
 galaxy_info:
-  author: First Last
+  author: Alexander King
   company: CISA Cyber Assessments
-  description: Skeleton Ansible role
+  description: This role installs the Admiral Python Library for Certificate Transparency log scanning.
```

Overzealous capitalization:
```suggestion
  description: This role installs the Admiral Python library for Certificate Transparency log scanning.
```